pkgs/tools/networking/connman/default.nix

```diff
@@ -84,8 +84,10 @@ stdenv.mkDerivation rec {
   ;
 
   # fix invalid path to 'file'
-  preConfigure = ''
+  patchPhase = ''
```

Comment:
Sorry, I actually meant to suggest using `postPatch`,  so you don't have to run the hooks manually.

Reply:
Fixed.